fix logical order of documentation Object Types (#2841)
microsoft/TypeScript-Website: packages/documentation/copy/en/handbook-v2/Object Types.md

}
```

> As discussed in the section index signatures, we can saying that `SquareConfig` can have any number of properties, and as long as they aren't `color` or `width`, their types don't matter.

Reviewer: ```suggestion
Here we're saying that `SquareConfig` can have any number of properties, and as long as they aren't `color` or `width`, their types don't matter.
```

This was just the section previously; this little comment was previously intended to be "don't worry about this syntax, we'll describe it later", but since it's already introduced in the current page, we can just not mention it because knowledge of it is assumed.
